vizro-ai/docs/pages/user-guides/add-generated-chart-usecase.md

```diff
@@ -23,23 +21,25 @@ This guide explains the different ways in which you can add a chart generated by
             df = px.data.gapminder()
             vizro_ai = VizroAI(model="gpt-4o")
 
-            result = vizro_ai.plot(df,
-              """Plot a bubble chart to show the changes in life expectancy
-               and GDP per capita for each country over time.
-               Color the bubbles by continent.
-               Add animation on yearly basis, and do not use facets.
-               Put the legend on top""", return_elements=True)
+            result = vizro_ai.plot(
+                df,
+                """Plot a bubble chart to show the changes in life expectancy
+                    and GDP per capita for each country over time.
+                    Color the bubbles by continent.
+                    Add animation on yearly basis, and do not use facets.
+                    Put the legend on top""",
+                return_elements=True,
+            )
 
-            print(f"Insight:\n{result.chart_insights}\n" )
-            print(f"Code explanation:\n{result.code_explanation}\n\nCode:\n{result.code_vizro}\n" )
+            print(f"Insight:\n{result.chart_insights}\n")
+            print(f"Code explanation:\n{result.code_explanation}\n\nCode:\n{result.code_vizro}\n")
             result.get_fig_object(df).show()
             ```
-        === "Result"
-            [![VizroAIChart]][VizroAIChart]
 
-        [VizroAIChart]: ../../assets/user_guides/vizro-ai-chart.png
+        === "Result"
+            [![VizroAIChart]][vizroaichart]
 
-2. Insert the resulting chart into a dashboard.
+1. Insert the resulting chart into a dashboard.
```

Comment:
I'm a bit confused now. As I understood the `vale` is removed and we use the `mdformat` now. 

1. When this hook is triggered? If it's triggered on every commit, how it's possible to fix this this line (from `2.` -> `1.`) as this action will revert all our manual changes.

2. And, what `<!-- vale off -->` and `<!-- vale on -->` now mean for us? Sorry for thousand questions

Reply:
Not a problem. So the numbering is absolutely fine to leave as 1. everywhere. It's just markdown automation, the numbers will come out correctly (if you [check the built docs](https://vizro--873.org.readthedocs.build/projects/vizro-ai/en/873/pages/user-guides/add-generated-chart-usecase/#use-vizro-ais-generated-code)).

Vale is going to stay in the toolkit but not as part of automated linting, we'll just use it as and when a big docs change happens, to manually catch any typos etc. So we need to retain the vale flags, which are added manually to turn vale off for certain things (like numbers in headings, which it hates).